### PR TITLE
Add 48 hr notice and standardise capitalization

### PR DESCRIPTION
**Pull request by LindsayYoung**
F6 was not listed in the form types section
standardized capitalization

### CODE REVIEW

File: webservices/decoders.py [hunk 1]
```diff
@@ -15,11 +15,12 @@
 
 form_types = {
     'F1': 'Statement of organization',
-    'F1M': 'Notification Of Multicandidate Status',
+    'F1M': 'Notification of multicandidate Status',
```
Comment (jmcarp) [inline webservices/decoders.py]: Should we also lower-case "Status"?

File: webservices/decoders.py [hunk 2]
```diff
@@ -15,11 +15,12 @@
...
     'F2': 'Statement of candidacy',
     'F9': '24-hour notice of disbursements for electioneering communications',
     'F10': '24-hour notice of expenditure of personal funds',
     'F11': '24-hour notice of opposition personal funds amount',
     'F12': '24-hour notice of suspension of increased limits',
-    'F99': 'Miscellaneous Document',
+    'F99': 'Miscellaneous document',
+    'F6': '48-Hour notice contribution or loans',
```
Comment (jmcarp) [inline webservices/decoders.py]: For consistency, how about lower-casing "Hour", and using the full title of the form: "48-hour notice of contributions/loans received"?

File: webservices/decoders.py (hunk 1 above)
Comment (LindsayYoung) [inline webservices/decoders.py]: Yea!

File: webservices/decoders.py (hunk 2 above)
Comment (LindsayYoung) [inline webservices/decoders.py]: sure, I will update those two

**Comment by LindsayYoung** [comment]
This looks like it already happened

**Pull request closed without merging**